nearmint/src/lib.rs

```diff
@@ -161,14 +162,26 @@ impl RuntimeAdapter for NearMint {
         )
     }
 
-    fn view_access_key(&self, account_id: &String) -> Result<Vec<PublicKey>, String> {
+    fn view_access_key(
+        &self,
+        account_id: &AccountId,
+        public_key: &PublicKey,
+    ) -> Result<Option<AccessKey>, String> {
+        let state_update = TrieUpdate::new(self.trie.clone(), self.root);
```

Comment:
(for my education): why do we need a state_update for a view only operation?

Reply:
So this is how we access the state from trie, even if you don't modify it.